### PR TITLE
Calendar front

### PR DESCRIPTION
**Pull request by camillaume**

### CODE REVIEW

**Review by ghost — DISMISSED** [review]
Okey for me

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

File: frontend/src/components/CalendarComponent/index.jsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  weekdaysParseExact: true,
+  longDateFormat: {
+    LT: "HH:mm",
+    LTS: "HH:mm:ss",
+    L: "DD/MM/YYYY",
+    LL: "D MMMM YYYY",
+    LLL: "D MMMM YYYY HH:mm",
+    LLLL: "dddd D MMMM YYYY",
+  },
+});
```
Comment (LoicBrassart) [inline frontend/src/components/CalendarComponent/index.jsx]: May I know why you'e rexriting all the FR locales here ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Remove unnecessary code (or explain to me why you need it)

File: frontend/src/components/CalendarComponent/index.jsx (hunk 1 above)
Comment (camillaume) [reply frontend/src/components/CalendarComponent/index.jsx]: Because without this code, moment is in english

**Review by ghost — APPROVED** [review]
Ok for me

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**